api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/MultiValueEditSubmission.java

```diff
@@ -304,9 +304,14 @@ public void addLiteralToForm(EditConfigurationVTwo editConfig, FieldVTwo field,
 					String rangeLang = field.getRangeLang();  //UQAM  Default value
 					try {
 						if (_vreq != null ) {
+							// only if the request comes from the rdfsLabelGenerator the language should be used
+							Boolean get_label_language = false;
+							if (!StringUtils.isBlank(editConfig.formUrl) && editConfig.formUrl.contains("RDFSLabelGenerator")) {
+								get_label_language = true;
```

Comment:
Please change variable name to camelCase: `getLabelLanguage`.

Reply:
done with commit 47196383fe1fefb5da843e985c4d2f5917d14696